test/test-cases/custom-src-paths/.gitignore

```diff
@@ -1,6 +1,7 @@
 # managed by sku
 .eslintrc
 .prettierrc
+coverage/
```

Comment:
Realise this has nothing to do with your PR but pretty weird it's showing in this PR.

Reply:
It's because the Renovate PR didn't run `sku configure`.

Reply:
This will go away once I rebase off the changes from https://github.com/seek-oss/sku/pull/253/

Reply:
This is now gone with merge from master